### DISCUSSION TITLE
Import FastAPI does not work.

### DISCUSSION THREAD

**Discussion by estkae** [fastapi/fastapi#6723] [Questions] [answered]
### First check

* [ ] I added a very descriptive title to this issue.
* [ ] I used the GitHub search to find a similar issue and didn't find it.
* [ ] I searched the FastAPI documentation, with the integrated search.
* [ ] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [ ] I already read and followed all the tutorial in the docs and didn't find an answer.
* [ ] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [ ] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [ ] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [ ] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### Example

Here's a self-contained, [minimal, reproducible, example](https://stackoverflow.com/help/minimal-reproducible-example) with my use case:

```Python
from fastapi import FastAPI

app = FastAPI()


@app.get("/")
def read_root():
    return {"Hello": "World"}
```

### Description

* Open the browser and call the endpoint `/`.
* It returns a JSON with `{"Hello": "World"}`.
* But I expected it to return `{"Hello": "Sara"}`.

### Environment

* OS: [e.g. Linux / Windows / macOS]:
* FastAPI Version [e.g. 0.3.0]:

To know the FastAPI version use:

```bash
python -c "import fastapi; print(fastapi.__version__)"
```

* Python version:

To know the Python version use:

```bash
python --version
```

**Comment by estkae** [comment]
if i start the example then i have this issue:
ImportError: cannot import name 'FastAPI' from partially initialized module 'fastapi' (most likely due to a circular import) (./fastapi.py)

**Comment by estkae** [comment]
Solved

**Comment by chbndrhnns** [comment, marked as answer]
Is it because your file was called `fastapi.py`?

**Comment by tiangolo** [comment]
Thanks for the help here @chbndrhnns ! 👏  🙇 

Thanks for coming back and closing the issue @estkae  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
